core/orchestrator.go

```diff
@@ -663,7 +663,7 @@ func (rt *RemoteTranscoder) Transcode(job string, fname string, profiles []ffmpe
 		return nil, RemoteTranscoderFatalError{err}
 	}
 
-	fullProfiles, err := common.FFmpegProfiletoNetProfile(profiles)
+	fullProfiles, _, err := common.FFmpegProfiletoNetProfile(profiles)
```

Comment:
Assuming for now that standalone Ts will be upgraded in tandem with Os, so we aren't attempting to handle back compat here.